css/dashboard/expand-students.less

```diff
@@ -0,0 +1,6 @@
+.expandStudents {
+  width: 120px;
+  position: absolute;
+  left: 30px;
+  top: 200px;
```

Comment:
Ideally the button would go in a cell next to the question row which expands/shrinks along with the questions. But, I just put it in a corner for now, since I wasn't sure if we would end up wanting 2 buttons anyway.